soc/xtensa/nxp_adsp/Kconfig

```diff
@@ -14,6 +14,10 @@ config SOC_FAMILY
 	string
 	default "nxp_adsp"
 
+config NXP_ADSP_SIGN_IMAGE
+	bool "Sign DSP image with rimage"
+	default y
+
 # Select SoC Part No. and configuration options
 source "soc/xtensa/nxp_adsp/*/Kconfig.soc"
```

Comment:
@dluke62 I wonder how big of an effort would be for you to split this PR into multiple patches. 

For example, I would like to see the introduction of NXP_ADSP_SIGN_IMAGE and preparation to accomodate RT in a separate patch.

Usually we do cleanups and preparation patches first and then add a big patch with the support for a new boards. 

What do you think?

Reply:
hi @dbaluta , thank you for your comment, I think i can split the PR in two commits - first one will update the Kconfigs in preparation for RT SoC, second one will add the board/soc files for RT

Reply:
@dluke62 thanks. That seems reasonable. Usually, we add one patch per logical change.